software/base/src/main/java/brooklyn/entity/software/MachineLifecycleEffectorTasks.java

```diff
@@ -614,6 +631,23 @@ public void stop(ConfigBag parameters) {
         if (log.isDebugEnabled()) log.debug("Stopped software process entity "+entity());
     }
 
+    @VisibleForTesting
+    public static boolean canStop(StopMode stopMode, boolean isEntityStopped) {
```

Comment:
May as well reduce the visibility of the method if it's only exposed for testing.